macros/src/main/scala/clue/macros/GraphQL.scala

```diff
@@ -48,38 +49,68 @@ private[clue] final class GraphQLImpl(val c: blackbox.Context) extends GraphQLMa
    * `parAccum` accumulates parameters until we have a whole case class definition.
    */
   private[this] case class ClassAccumulator(
-    classes:  List[CaseClass] = List.empty,
-    parAccum: List[ClassParam] = List.empty
-  )
+    classes:  List[Class] = List.empty,
+    parAccum: List[ClassParam] = List.empty,
+    sum:      Option[Sum] = None
+  )                                     {
+    def withOverrideParams: ClassAccumulator =
+      copy(parAccum = parAccum.map(_.copy(overrides = true)))
+  }
+  private[this] object ClassAccumulator {
+    implicit val monoidClassAccumulator: Monoid[ClassAccumulator] = new Monoid[ClassAccumulator] {
```

Comment:
Do we have a monoid law check for this?
I'm only asking this because I'm unqualified to ask smarter questions.

Reply:
Good catch. I had this pending. I'll move it out the macro code and test laws.